project/Dependencies.scala

```diff
@@ -51,6 +52,7 @@ object Dependencies {
     "com.univocity"                  % "univocity-parsers"   % "2.4.1",
     "org.ocpsoft.prettytime"         % "prettytime"          % "4.0.1.Final",
     "org.everit.json"                % "org.everit.json.schema" % "1.4.1",
+    "com.github.pathikrit"          %% "better-files"        % "2.17.1",
```

Comment:
3.0.0 isn't in maven yet: https://mvnrepository.com/artifact/com.github.pathikrit